opensearch/sycamore-opensearch.sh

```diff
@@ -422,9 +498,34 @@ sp_create_non_rag_pipeline() {
 END
 }
 
+flick_rag_feature() {
+    # This exists because the rag_pipeline_feature_enabled setting
+    # only recognizes when it changes. Since it starts in the 'on'
+    # position now, we have to turn it off and then on again
+    _curl_json -X PUT "${BASE_URL}/_cluster/settings" \
```

Comment:
I should probably open an issue to fix this in ml-commons?

Reply:
if you would be so kind. Sorry, I should have caught this when I turned the feature flag on